src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/PickupCartHandler.php

```diff
@@ -37,6 +37,7 @@ public function __construct(
         private ObjectManager $orderManager,
         private RandomnessGeneratorInterface $generator,
         private CustomerRepositoryInterface $customerRepository,
+        private int $tokenLength = 10,
```

Comment:
Why not `64` by default here as well?
```suggestion
        private int $tokenLength = 64,
```

Reply:
I see new parameter `sylius_core.order_token_length` introduced, but I don't see any usage of it.

It needs to be injected to `Sylius\Bundle\ApiBundle\CommandHandler\Cart\PickupCartHandler` and `Sylius\Component\Core\TokenAssigner\UniqueIdBasedOrderTokenAssigner` services?

Reply:
Yeah, the commit setting these apparently poofed somehow.